tallycash/extension
# Spot Check: Clean up some unused code, imports, and add names to activities

Quick and easy.

Activity list improvements are pretty short:
- Contract interactions show the To address.
- The activity list shows the name when available for the displayed
  address. An OS tooltip shows the full address.
- Activity details show name + address when the name is available.
  The existing click-to-copy functionality remains.

## Testing

Look at some existing account history and check that:
- Any addresses that can be resolved display as names in
  the list and name + address in the details.
- Make sure OS tooltips exist with the full address.
- Make sure the details address/name can be clicked to copy.
- Make sure contract interactions show the To instead of From
  address.
- Make sure token and ETH sends to other addresses still show
  the To instead of From.
- Make sure token/ETH receives show the From instead of To.

Receipts:

<img width="373" alt="Screen Shot 2022-04-01 at 17 17 06" src="https://user-images.githubusercontent.com/8245/161342398-95ae7435-141d-4ee2-bbcd-8dd9141847cd.png">
<img width="373" alt="Screen Shot 2022-04-01 at 17 17 45" src="https://user-images.githubusercontent.com/8245/161342483-aa7ca455-57d1-40e0-af6c-a95a175b719a.png">
<img width="475" alt="Screen Shot 2022-04-01 at 17 18 00" src="https://user-images.githubusercontent.com/8245/161342513-13b98440-3321-4a90-aa35-20c0737245e9.png">
